docs/appendices/release-notes/5.10.3.rst

```diff
@@ -59,3 +59,6 @@ Fixes
   :ref:`ADD COLUMN <sql-alter-table-add-column>`. The fix doesn't apply for
   columns which were already added to tables after their creation before
   :ref:`version_5.10.3`. Those columns need to be dropped and recreated.
+
+- Fixed an issue that caused selecting from partitioned tables created before
+  ``5.5.0`` to return nulls or ``oids`` as the column names.
```

Comment:
Please clarify the sentence, to say that they return ``NULL``s when returned from a `SELECT` query and then a sentence about the oid instead of a column name.

Reply:
Do you mean add two entries?